Users/Docker_Image.rst

```diff
@@ -63,7 +64,8 @@ on your code with a ``.travis.yml``, like this:
 
     language: generic
     services: docker
-    script: docker run --volume=$(pwd):/app --workdir=/app coala/base coala-ci
+    script: docker run --volume=$(pwd):/app --workdir=/app coala/base \
+            coala --ci
```

Comment:
I'm not fully sure this works with the line break, is this tested or can somebody with travis/better yml knowledge approve this? Otherwise this commit is good and I'd merge it :)